frontend/js/src/explore/fresh-releases/components/ReleaseTimeline.tsx

```diff
@@ -15,7 +18,7 @@ function createMarks(
   sortDirection: string,
   order: string
 ) {
-  let dataArr: Array<string> = [];
+  let dataArr: Array<React.ReactNode> = [];
```

Comment:
This might be more appropriate:
```suggestion
  let dataArr: Array<string | JSX.Element> = [];
```